SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/EventTracerComponent.cpp

```diff
@@ -0,0 +1,78 @@
+#include "EventTracerComponent.h"
```

Comment:
We had this as part of the project. Moving to GDK so we can test ET in functional tests.